components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2038,6 +2038,12 @@ private Condition getCondition(Node node, Map<String, String> attributes) throws
                 conditionOperation = ExpressionOperation.GE.toString();
             } else if (SCIMCommonConstants.LE.equals(operation)) {
                 conditionOperation = ExpressionOperation.LE.toString();
+            } else if (SCIMCommonConstants.GT.equals(operation)) {
+                conditionOperation = ExpressionOperation.GT.toString();
```

Comment:
Do we support these operations from the user store level. If not what would be the behavior if you use this operation in filtering?

Reply:
Yes, supported these operations for LDAP User stores and JDBC User stores